docs/source/markdown/podman-save.1.md

```diff
@@ -27,7 +27,7 @@ Note: `:` is a restricted character and cannot be part of the file name.
 #### **--compress**
 
 Compress tarball image layers when pushing to a directory using the 'dir' transport. (default is same compression type, compressed or uncompressed, as source)
-Note: This flag can only be set with **--format=docker-dir**.
+Note: This flag can only be set with **--format=docker-dir** or **--format=oci-dir**.
```

Comment:
This is consistent with the check at the CLI handler level, but AFAICS not actually correct: The option drives `ImageSaveOptions.Compress`, which turns into `libimage.CopyOptions.DirForceCompress`, and that doesn’t affect `oci-dir`.

So I think it’s the CLI validation that should be changed.

Reply:
@mtrmac Thank you for the feedback. I made changes, let me know if this is what you're looking for.